packages/core/src/utils/cloudflare/types.ts

```diff
@@ -4,3 +4,9 @@ export const cloudflareResponseGuard = z.object({
   success: z.boolean(),
   result: z.unknown(),
 });
+
+export const cloudflareHostnameResponseGuard = z
+  .object({
+    origin: z.string(),
+  })
+  .catchall(z.unknown());
```

Comment:
no need for `catchall()` since be default zod will ignore those unknown keys? (if ok it also applies to other guards like `cloudflareDataGuard`)

Reply:
created LOG-6302 to track, will do it in the cooldown cycle.